include/libbase64.h

```diff
@@ -18,6 +18,7 @@ extern "C" {
 
 struct base64_state {
 	int eof;
+	int aeof;
```

Comment:
yes, could be done.
I guess that structure is for internal use anyway. Why isn't it opaque structure ? To avoid `malloc`/ `free` inside the lib ?